### PR TITLE
added Authors and link to blog

### PR DESCRIPTION
**Pull request by hannahkane**
per https://github.com/mozilla/learning.mozilla.org/issues/2477

### CODE REVIEW

File: web-lit-whitepaper/index.html [hunk 1]
```diff
@@ -29,6 +29,7 @@
     <article id="introduction">
 
       <h1>Web Literacy 2.0</h1>
+      <h2> Authors: Authors: An-Me Chung, Iris Bond Gill, Ian O'Byrne</h2>
```
Comment (flukeout) [inline web-lit-whitepaper/index.html:32]: @hannahkane This says Authors: Authors:

**Review by flukeout — CHANGES_REQUESTED** [review]

**Comment by flukeout** [comment]
Also changing the markup to...

``<p><strong>Authors:</strong> An-Me Chung, Iris Bond Gill, Ian O'Byrne</p>``

Produces this...



Which I think is better and not so crazy big.

File: web-lit-whitepaper/index.html (hunk 1 above)
Comment (hannahkane) [reply web-lit-whitepaper/index.html:32]: Maybe one of the authors names is Authors. Did you think about that? JK. Will fix.

Comment (flukeout) [reply web-lit-whitepaper/index.html:32]: Also stray space right after the ``<h2>`` tag

**Comment by flukeout** [comment]
cc @hannahkane left some comments

**Pull request closed without merging**